lib/web/mage/calendar.js

```diff
@@ -377,6 +377,9 @@
                 .addClass('v-middle')
                 .text('') // Remove jQuery UI datepicker generated image
                 .append('<span>' + pickerButtonText + '</span>');
+
+            $(element).attr('autocomplete', 'off');
```

Comment:
Hi @vasilii-b. Thanks for the collaboration. This fix removed autocomplete in all scenarios. Are you sure that is the right approach?

Reply:
I will as product manager, because it more related to the produce manager that to developer. 

Anyway, I think we should add the configuration option to manage this situation and provide a choice to using this component in each case

Reply:
@VladimirZaets sounds reasonable. I'll add the additional configuration.
Thank you!